README.md

```diff
@@ -170,6 +170,7 @@ installation and/or support for using Julia to its users:
 [HLRS](https://www.hlrs.de) | [Hawk](https://www.hlrs.de/systems/hpe-apollo-hawk/) | ✅ | ✅ | ✅ | [AMD EPYC Rome](https://www.hlrs.de/systems/hpe-apollo-hawk/) | [Nvidia Tesla A100](https://www.hlrs.de/systems/hpe-apollo-hawk/) | [1](https://kb.hlrs.de/platforms/index.php/Julia)
 [HPC @ LLNL](https://hpc.llnl.gov/) | various systems | ✅ | ? | ✅ | various processors | various GPUs | [1](https://hpc.llnl.gov/services/jupyter/julia)
 [HPC2N, Umeå U](https://www.hpc2n.umu.se/) | [Kebnekaise](https://www.hpc2n.umu.se/resources/hardware/kebnekaise) | ✅ | ✅ | ? | [Intel Xeon Broadwell + Skylake](https://www.hpc2n.umu.se/resources/hardware/kebnekaise) | [Nvidia Tesla K80](https://www.hpc2n.umu.se/resources/hardware/kebnekaise), [Nvidia Tesla V100](https://www.hpc2n.umu.se/resources/hardware/kebnekaise) | [1](https://www.hpc2n.umu.se/resources/software/julia)
+[LuxProvide](https://luxprovide.lu/) (EuroHPC) | [MeluXina](https://luxprovide.lu/#meluxina) | ✅ | ? | ✅ | [AMD EPYC Rome](https://docs.lxp.lu/system/overview/) | [Nvidia Ampere A100-40](https://docs.lxp.lu/system/overview/) | [1](https://docs.lxp.lu/first-steps/software_env/)
```

Comment:
```suggestion
[LuxProvide](https://luxprovide.lu/) (EuroHPC) | [MeluXina](https://luxprovide.lu/#meluxina) | ✅ | ? | ✅ | [AMD EPYC Rome](https://docs.lxp.lu/system/overview/) | [Nvidia Ampere A100-40](https://docs.lxp.lu/system/overview/) | [1](https://docs.lxp.lu/software/module_example/julia/), [2](https://docs.lxp.lu/first-steps/software_env/)
```
I suggest to put the Julia-specific link to the docs first, and the general software link second